mesheryctl/pkg/utils/healthcheck.go

```diff
@@ -114,6 +115,12 @@ func parseKubectlShortVersion(version string) ([3]int, error) {
 
 // IsMesheryRunning checks if the meshery server containers are up and running
 func IsMesheryRunning(currPlatform string) (bool, error) {
+	// Checking if Meshery is running with "make run-local" or "make run-fast"
+	resp, _ := http.Get("http://localhost:9081/api/server/version")
```

Comment:
The hostname and port cannot be hardcoded, but must use a central reference for these values.

Reply:
@leecalcote Check here https://github.com/meshery/meshery/blob/master/Makefile#L66

Reply:
@leecalcote that's what I felt when I saw this, on public version, the port number may tend to change depending on user config. But here we check locally using `make run-local` or `make run-fast` (which works only on port 9081 or desired port)

IDK if it's ok for now

Reply:
Users don't deploy using the Makefile. The hostname and port cannot be hardcoded, but must use a central reference for these values.

Reply:
Agreed, so à placeholder variable then in this case?

Reply:
@leecalcote this is being added for a specific case i.e. Development Mode of Meshery Server.

If some developer made some changes in server-side and is testing out a mesheryctl command (e.g. `mesheryctl system login`) which needs some confirmation that IsMesheryRunning(), which will eventually fail because of the only platform based checking and it won't possible to test the command with new changes in server. that is why the hardcoded part.

Reply:
So..... are we good to go or still needs to be looked on this?

Reply:
Unless the hardcoded value used in this check only applies to developers who are compiling `mesheryctl` and is not enforced for users who are running Meshery at an unknown endpoint (a dynamically defined endpoint), no, we cannot proceed until hardcoded `localhost` is removed.

Reply:
OK........... Makes sense at first look at the problem. So the dynamic way then :)